### DISCUSSION TITLE
AttributeError: 'Config' object has no attribute 'omnistaging_enabled'

### DISCUSSION THREAD

**Discussion by emiryolcu99** [netket/netket#1298] [Q&A] [answered]
Hello everyone. At now I'm working on Ubuntu 22.04 LTS. When I was writing "import netket" I got this output:

`import netket`

> 2022-08-18 19:54:09.851827: W external/org_tensorflow/tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory

>2022-08-18 19:54:09.877472: W external/org_tensorflow/tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory

>2022-08-18 19:54:09.879111: W external/org_tensorflow/tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory

Traceback most recent call last:
  File "stdin", line 1, in module
  File "/home/emir/anaconda3/envs/netket/lib/python3.8/site-packages/netket/__init__.py", line 24, in <module>
    from . import utils
  File "/home/emir/anaconda3/envs/netket/lib/python3.8/site-packages/netket/utils/__init__.py", line 38, in <module>
    from . import mpi
  File "/home/emir/anaconda3/envs/netket/lib/python3.8/site-packages/netket/utils/mpi/__init__.py", line 15, in <module>
    from .mpi import (
  File "/home/emir/anaconda3/envs/netket/lib/python3.8/site-packages/netket/utils/mpi/mpi.py", line 44, in <module>
    if not jax.config.omnistaging_enabled:
AttributeError: 'Config' object has no attribute 'omnistaging_enabled'

What should I do? I'm always upgrading `pip` before writing and I updated the `jax` and `mpi`.

**Comment by PhilipVinc** [comment, marked as answer]
Your jax version is too recent. You must either downgrade, or wait a few hours and I'll release a new version of netket that works

**Reply by emiryolcu99** [reply]
Thanks for your answer Philip. I think I can wait a bit.

**Reply by PhilipVinc** [reply]
I just tagged 3.5, you can update netket and it will work with the latest version of jax
